DeviceSimulator/DeviceSimulator/IotHubConnection.cs

```diff
@@ -0,0 +1,302 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I copied this file from https://github.com/Azure/azure-sdk-for-net/tree/master/samples/iothub-connect-to-eventhubs.

I talked with someone from the EventHubs team. They don't provide a ctor for IoT Hub connection string any more. Rather they expect you to get the event hub-compatible connection string from your hub. Unfortunately, there's no supported programmatic way to do that right now.

This sample file works though, so we'll use it, so our user doesn't have to provide yet another connection string.

Meanwhile, I'm planting the idea that our track 2 IoT Hub service SDK should support getting this property.